src/Style/General/AlertStyle.ts

```diff
@@ -43,7 +43,7 @@ export const AlertContainer = styled.div<AlertContainerProps>`
   font-size: 1em;
 
   @media ${Device.mobileM} {
-    min-width: 100%;
+    min-width: 95%;
```

Comment:
@ritikamotwani do you think this is better? That we keep the spacing 15px on both side.
```
min-width: calc(100% - 30px);
```

Reply:
Thanks @tingtingdisco 
Updated this :smile: